api/APIREADME.md

```diff
@@ -0,0 +1,112 @@
+# CoTripper API
+
+## Description
+
+CoTripper is a Django-based project aimed at providing an experience-based travel platform for single mom families. CoTripper aims to encourage ideas, exposure, and to be helpful around helping planning vacations and building community. We're more than a travel solution.
+
+## Technologies
+
+- Python
+- Django
+- PostgreSQL
+- Psql
+- Django Rest Framework
+
+## Database Setup
+
+1. Clone the Repository
+
+```
+git clone git@github.com:CotripperPlatform/CoTrip.git
+```
+
+2. Install pipenv and then install psycopg2-binary and autopep8
+
+```
+pipenv install
+
+pipenv install psycopg2-binary
+
+ pipenv install autopep8
+```
+
+3. Replace the DATABASE dictionary in the settings.py within the project directory with the following code:
+
+```
+# settings.py
+
+DATABASES = {
+   'default': {
+       'ENGINE': 'django.db.backends.postgresql',
+       'NAME': 'cotripper',
+       'USER': 'cotripperuser',
+       'PASSWORD': 'cotripper',
+       'HOST': 'localhost'
+   }
+}
+```
+
+4. Create settings.sql file in CoTrip/api:
+
+```
+touch settings.sql
+
+```
+
+5. Within the settings.sql file include the following text:
+
+```
+CREATE DATABASE cotripper;
+CREATE USER cotripperuser WITH PASSWORD 'cotripper';
+ GRANT ALL PRIVILEGES ON DATABASE cotripper TO cotripperuser;
+
+```
```

Comment:
Check with @tissakhosla on this - I think he's including a `settings.sql` file and the database configuration for local development, so all they should have to do is load the sql file into Postgres. Something like:

```sh
psql -u settings.sql
```

Though double check that.

👏

Reply:
You have it on line 74!